src/coreclr/jit/optimizer.cpp

```diff
@@ -5958,60 +5958,26 @@ bool Compiler::optIsSetAssgLoop(unsigned lnum, ALLVARSET_VALARG_TP vars, varRefK
         return true;
     }
 
-    switch (loop->lpAsgCall)
+    // If caller is worried about possible indirect effects, check
+    // what we know about the calls in the loop.
+    //
+    if (inds != 0)
```

Comment:
Are any of the callers worried?

(I do not see us pass anything but `VR_NONE` for `inds`).

Reply:
You're right, nobody actually cares right now. And there are other parts of the analysis that are likewise unused.

I'm deliberately leaving it for now. It's possible we'll run across cases where we want to try and use it for something.